config/routes.php

```diff
@@ -0,0 +1,6 @@
+<?php
+use Cake\Routing\Router;
+
+Router::plugin('CreatorModifier', function ($routes) {
+	$routes->fallbacks('InflectedRoute');
+});
```

Comment:
Do we need to define routes for a behavior-only plugin? Is this required for something I'm not thinking of?